setup.sh

```diff
@@ -22,8 +22,10 @@ function help() {
     echo -e "\tPFUNIT_DIR        The directory where PFUnit is installed"
     echo -e "\tGSLIB_DIR         The directory where GSLIB is installed"
     echo -e "\tCUDA_DIR          The directory where CUDA is installed"
+    echo -e "\tHIP_DIR           The directory where HIP is installed"
     echo -e "\tBLAS_DIR          The directory where BLAS is installed"
     echo -e "\tCMAKE_VARIABLES   Additional variables to pass to CMake"
+    echo -e "\tEXTRA_FEATURES    Additional features to pass to neko configure"
```

Comment:
Please explicitly state that this is not used anywhere by us.
```suggestion
    echo -e "\tNEKO_FEATURES    Additional features to pass to neko configure"
```